lib/daemon.js

```diff
@@ -36,21 +37,38 @@ const commandsToFunctionNames = {
   add: 'addImports',
 };
 
+commander.option('--parent-pid <n>', parseInt).parse(process.argv);
+
 export default function daemon() {
   // The `importjsd` here is mostly a dummy file because config relies on a
   // `pathToCurrentFile`. Normally, this is the javascript file you are editing.
   const config = new Configuration('importjsd');
 
   loglevel.setLevel(config.get('logLevel'));
   loglevelMessagePrefix(loglevel);
-  originalConsoleLog(`DAEMON active. Logs will go to: \n${pathToLogFile}`);
+  originalConsoleLog(`DAEMON active. Logs will go to: ${pathToLogFile}`);
```

Comment:
It might be nice to throw in `ImportJS ${VERSION}` at the beginning of this as well.